docs/docs/community/team-structure.md

```diff
@@ -0,0 +1,61 @@
+---
+
+The Nebari project has a team-based governance structure, implemented transparently using [GitHub's team features](https://github.com/orgs/nebari-dev/teams).
+This page documents the different teams, their privileges and responsibilities, and how you can join the teams.
+
+:::tip
+Use the table of contents in the right sidebar to jump to different sections of this page.
+:::
+
+## Teams
+
+The following teams are formally recognized by Nebari.
+
+### Triage
+
+Members responsible for [triaging](./maintainers/triage-guidelines) and [improving](./file-issues#working-on-issues-to-improve-them) new and existing issues & pull requests (PRs) across all repositories. They can add relevant labels to issues/PRs, edit issue/PR titles and descriptions to improve them, and transfer issues between repositories.
+
+### Contributors
+
+People who make valuable contributions to the Nebari projects in the form of code contributions, documentation improvements, or design enhancements, issues/PR triage, community management, fundraising, advocacy, and more. Members of this team have commit rights across all Nebari repositories, which includes access to [triage](#triage) permissions.
+
+### Core
+
+Project sustainers who lead and maintain the overall Nebari project and community. They are the core and final decision-makers on project-wide initiatives, and have repository-level management rights across all Nebari projects.
+
+### Owners
+
+A private and small subset of the Core team who have organization-level privileges for [`nebari-dev`](https://github.com/nebari-dev). While decisions are made by the entire Core team, the Owners group has relevant permissions to implement certain decision like adding new team members.
+
+### Conduct
+
+Community members responsible for upholding and enforcing Nebari's Code of Conduct. This team is documented in the [Nebari governance repository](https://github.com/nebari-dev/governance/blob/main/code-of-conduct/coc_enforcement.md#the-code-of-conduct-committee). At least one member of this team should also be in the Nebari Core team, and all members should be trained to handle CoC reports with care.
+
+### Emeritus core
+
+The Emeritus core team recognizes community members who choose to step away from the Core team. Open source projects have members join and retire as the project matures. Contributors can move on for various reasons including limited bandwidth, conflicting or changing interests, burnout, boredom, personal reasons, and much more.
+
+Nebari Core team members can step away at any time, and we hope for them to communicate this to the rest of the Core team early when possible. They can share as much or little details as they are comfortable sharing for this move. We also hope they can work with the team for a smooth transition, but we do understand if sometimes this may not be possible. Fellow core team members are expected to support off-boarding member for a graceful transfer of responsibilities.
```

Comment:
```suggestion
Nebari Core team members can step away at any time, and we hope for them to communicate this to the rest of the Core team early when possible. They can share as much or as little details as they are comfortable sharing for this move. We also hope they can work with the team for a smooth transition, but we do understand if sometimes this may not be possible. Fellow core team members are expected to support off-boarding members for a graceful transfer of responsibilities.
```

Reply:
I's also suggest wording on they can change their status from emeritus to active too